datasets/nighttime-lights.data.mdx

```diff
@@ -11,7 +11,11 @@ media:
 taxonomy:
   - name: Topics
     values:
-      - Covid 19
+      - COVID 19
+  - name: Source
+    values:
+      - Black Marble
```

Comment:
this isn't the best `source` need something better.

Reply:
What would you suggest? We struggled to find a good source tag for this.